packages/utilities/psammead-locales/src/numerals/index.test.js

```diff
@@ -1,10 +1,46 @@
 import * as numerals from '.';
 
-describe('Numerals', () => {
+describe('Numeral systems', () => {
   const numeralSystems = Object.keys(numerals);
   numeralSystems.forEach(numeralSystem => {
     it(`should return ${numeralSystem} numerals`, () => {
       expect(numerals[numeralSystem]).toMatchSnapshot();
     });
   });
 });
+
+describe('makeNumeralTranslator', () => {
+  it('should return a function', () => {
+    const translate = numerals.makeNumeralTranslator();
+    expect(typeof translate).toBe('function');
+  });
+
+  describe('valid numeral system', () => {
+    const RomanNumerals = 'O_Ⅰ_Ⅱ_Ⅲ_Ⅳ_Ⅴ_Ⅵ_Ⅶ_Ⅷ_Ⅸ'.split('_');
+    const examples = [
+      ['0 1 2 3', 'O Ⅰ Ⅱ Ⅲ'],
+      ['9', 'Ⅸ'],
+      ['The Magnificent 7', 'The Magnificent Ⅶ'],
+    ];
+    const translate = numerals.makeNumeralTranslator(RomanNumerals);
+
+    it.each(examples)('should translate %s', (input, expected) => {
+      expect(translate(input)).toBe(expected);
+    });
+  });
+
+  it('invalid numeral systems', () => {
```

Comment:
maybe this should be a `describe` block having
```
it.each(invalidSystems)('should return ...
```
or still an it block with `invalid numeral systems should return something`

Reply:
I did this originally but I thought it was unhelpful to list all the invalid systems in the test output so I removed it.

Reply:
I've wrapped it in a describe block as you suggest so the output is a bit more descriptive, now it's:

```
  makeNumeralTranslator
    invalid numeral systems
      ✓ should return the input unchanged
```

Please take a look.